src/FsAutoComplete.BackgroundServices/Program.fs

```diff
@@ -469,9 +469,10 @@ type BackgroundServiceServer(state: State, client: FsacClient) =
             return LspResult.success ()
         }
 
-    member __.InitWorkspace() =
```

Comment:
instead of workspace context location being passed in during start (which doesn't pass any data to the launched background process), doing it during initworkspace makes a ton more sense to me.